src/components/themed/TransactionRow.tsx

```diff
@@ -85,10 +86,11 @@ const TransactionRowComponent = (props: Props) => {
   const pendingStyle = currentConfirmations === 'confirmed' ? styles.completedTime : styles.partialTime
 
   // Transaction Category
+  const defaultCategory = !isSentTransaction ? 'income' : 'expense'
```

Comment:
This default category logic also exists in the TransactionDetailsScene on line 67. You should copy the fancy `isSentTransaction` logic there so the two scenes agree. Otherwise QA will send us a bug where a transaction with an `undefined` category says "Income" in the transaction list but "Expense" in the transaction details scene.